### DISCUSSION TITLE
Next steps after deploying two EDCs

### DISCUSSION THREAD

**Discussion by neischlah** [eclipse-tractusx/tractusx-edc#515] [Q&A] [answered]
Hi everyone!

I followed [the example](https://github.com/eclipse-tractusx/tractusx-edc/tree/main/docs/samples/example-dataspace) and created two EDCs. My goal is to have an environment where I can implement a custom dataplane source, replacing a Postgresql instance in the example:
|EDC Sokrates| EDC Plato|
|--------|--------|
| Postgresql | ~~Postgresql~~ My own data source | 

1. Is the EDC from this repo a good start for my use case or am I better off deploying Eclipse's EDC for Plato?
2. Is there a documentation and/or example implementation for my use case?
3. How can I transfer data between the EDCs after completing the example? I also had a look at the e2e repo, but I can't find the actual calls. #514 may be related.

Thanks for your time!

**Comment by stephanbcbauer** [comment]
@neischlah for point 3 the postman collection is maybe helpful https://github.com/eclipse-tractusx/tractusx-edc/blob/main/docs/development/postman/collection.json

**Comment by stephanbcbauer** [comment]
@neischlah were you already able to test something? Which documentation was helpful? What was missing? THX

**Comment by matbmoser** [comment, marked as answer]
@neischlah my recomendation here is to deploy the open source digital product pass consumer application and retrieve data using it! You can register an aspect in your infrastructure, retrieve data from your edcs and display it in a cool UI.

You can learn then how to link data in your digital twin registry, data service and other important Catena-X Discovery Services.
We provide the charts for the data consumer and data provider parts: https://github.com/eclipse-tractusx/digital-product-pass/tree/main/deployment/infrastructure 

[Digital Product Pass Repo](https://github.com/eclipse-tractusx/digital-product-pass)

You can see how the basic architecture works:
![DPP Architecture](https://raw.githubusercontent.com/eclipse-tractusx/digital-product-pass/main/docs/arc42/media/dataRetrieval/digitalProductPassContext.drawio.svg)

Here is a screenshot:

![DPP Frontend](https://github.com/eclipse-tractusx/digital-product-pass/blob/main/docs/arc42/media/GraphicBatteryPassportViewGeneralInfo.png?raw=true)


You also have a postman collection for retrieving data using the EDC and the dpp-backend component: https://github.com/eclipse-tractusx/digital-product-pass/blob/main/deployment/local/postman/Digital-Product-Pass-collection.json

The dpp-backend component will retrieve any data aspect that you register in the EDC and in the digital twin registry. 

If you are interested, join our the second tractus-x community days event to learn how to handle with the Digital Product Pass.
